util/util.go

```diff
@@ -254,3 +255,19 @@ func IsRunningSystemd() bool {
 	}
 	return fi.IsDir()
 }
+
+// GetMachineID returns a host's 128-bit machine ID as a string.
+func GetMachineID() (machineID string, err error) {
+	// There is a systemd library function, sd_id128_get_machine, which can be used
+	// to obtain the machine ID.  However, that function simply reads the string
+	// from /etc/machine-id on the filesystem. To simplify things, we'll just read
+	// it directly from the file, too.
+	systemID, err := ioutil.ReadFile("/etc/machine-id")
+	if err != nil {
+		fmt.Errorf("Failed to read /etc/machine-id: %v", err)
```

Comment:
This isn't actually doing something, `fmt.Errorf` needs to be assigned (but per my comment about named result parameters let's just return)